particles/Tutorial/Kotlin/README.md

```diff
@@ -0,0 +1,106 @@
+
+>- *Particle* - Modular component of functionality. Ideally small units so particles can be interchangeable. 
+>- *Recipe* - A combination of particles to create features and systems.
+
+Particles and recipes are defined using the Arcs Manifest Language and implemented in Javascript, Kotlin, or C++.  The best way to understand this is to jump into some code. Let’s look at how Hello World is implemented.
+
+To start off with, we’re going to need two files, an Arcs manifest file (.arcs file) and a particle implementation in your language of choice. It’s easiest if we create a folder to house these files so we don’t lose them. As a matter of convention, particles live in the arcs/particles folder. Go ahead and create a new folder in there called ‘HelloWorld’.
+
+Don’t worry if you don’t understand everything at the moment, we’ll be going over all of this in more detail throughout this tutorial. Let’s walk through these files line by line, starting with the .arcs file. You should create this file in your folder. By convention, .arcs files are camel case, so it should be called `HelloWorld.arcs` or something similar.
+```
+// The file begins by defining our particle. Based on the file
+// path, We can assume the particle is implemented in kotlin.
+particle HelloWorldParticle in ''https://$arcs/bazel-bin/particles/Tutorial/Kotlin/hello_world.wasm''
+  // Notice the tab that starts this line. Whitespace matters
+  // in the Arcs manifest language, so this is very important.
+  // Don't worry about what this line does at the moment, we'll
+  // be getting to root and slots in more detail soon.
+  consume root
+
+// And now we are at the recipe definition!
+recipe HelloWorldRecipe
+  // Once again, notice the tab. This line tells us that the
+  // HelloWorldRecipe contains a HelloWorldParticle.
+  HelloWorldParticle
+  // Finally, we provide a human readable description for the
+  // recipe. This is optional, but will make it much easier and
+  // nicer to debug. Once again, notice the tab.
+  description `Tutorial 1: Hello World`
+```
+
+Alright, we’ve got our Arcs manifest file set. Now onto the Kotlin. Just like the .arcs file, this should be in your HelloWorld folder. We set the file name in the .arcs file above to be `HelloWorld.kt`.
+
+```kotlin
+// We start with the required imports and package info
+package arcs
+import kotlin.native.internal.ExportForCppRuntime
+
+// We define the HelloWorldParticle class to be a child
+// of the Particle class.
+class HelloWorldParticle : Particle() {
+
+  // For the moment, don't worry about this function. We will be explaining
+  // handles, slots, and root in plenty of details in upcoming tutorials.
+  override fun onHandleUpdate(handle: Handle) {
+    renderSlot("root")
+  }
+
+  // Again, don't worry about this function. We promise you'll understand
+  // what it does soon enough.
+  override fun onHandleSync(handle: Handle, allSynced: Boolean) {
+    if(allSynced) {
+      log("All handles synched\n")
```

Comment:
Why teach people to put logging statements in their code? I presume it is a leftover of us debugging the particle, but it's not something we should encourage.